spec/spec_helper.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+#
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+RSpec.configure do |config|
```

Comment:
- [ ] I am putting this comment here because the file the previous reviewer made the comment has been deleted, in the previous review, some of your tests failed and the reviewer told you to fix it, but you went and deleted the files, it is very important we understand why we write tests, this is to make sure our code works properly, kindly add back those files and try fixing the issues, :+1: